pages/community/styleguide/code-examples.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Code examples
+weight: 20
+---
+
+# Code Examples
+
+## Guidelines
```

Comment:
```suggestion
```

It doesn't make sense to have an h1 with no content, so let's delete this superfluous header. This copy seems a better intro summary to the topic, anyway.